### DISCUSSION TITLE
ETA for Xamarin.iOS 16.1 for macos-12 image

### DISCUSSION THREAD

**Discussion by akaegi** [actions/virtual-environments#6968] [Image configuration]
Hello,
do you have an estimate when Xamarin.iOS 16.1 will be available with the macos-12 image?
Unfortunately my app does not build with 16.0 Xamarin.iOS SDK, so I'm eagerly waiting until 16.1 will be available with the macos-12 image.
Many thanks!

**Comment by mikhailkoliada** [comment]
Hello! We are working with the responsible team to getting this version added as soon as possible!

**Comment by mikhailkoliada** [comment]
Hello. New Xamarin.Ios has been committed into main, so it will be available with the next image release
